src/profile.c

```diff
@@ -1270,6 +1298,10 @@ profile_transcode_can_share(profile_chain_t *prch,
     return 0;
   if (profile_transcode_bandwidth(pro1) != profile_transcode_bandwidth(pro2))
```

Comment:
Note that vbitrate/abitrate makes the bandwidth useless. Perhaps, it's time to remove it, because it's not used (and was prepared for exactly same thing).